src/main/java/ch/wisv/payments/rest/WebhookServiceImpl.java

```diff
@@ -0,0 +1,47 @@
+package ch.wisv.payments.rest;
+
+import ch.wisv.payments.model.Order;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.http.NameValuePair;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.entity.UrlEncodedFormEntity;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.message.BasicNameValuePair;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
+public class WebhookServiceImpl implements WebhookService {
+
+    /**
+     * Notify ... of a OrderStatus change.
+     *
+     * @param order of type Order
+     */
+    @Override
+    public void sendOrderStatusChangeNotification(Order order) {
+        // Skip if no webhook url is set.
+        if (order.getWebhookUrl() == null) {
+            return;
+        }
+
+        try {
+            HttpClient httpclient = HttpClients.createDefault();
```

Comment:
Is there a clear advantage over using the Spring RestTemplate for this operation? https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/client/RestTemplate.html

Reply:
The advantage of me knowing about it, however think it's beter to use the RestTemplate. So I will change it